java/driver/jdbc-validation-postgresql/src/test/java/org/apache/arrow/adbc/driver/jdbc/postgresql/PostgreSqlTypeTest.java

```diff
@@ -85,27 +85,27 @@ protected void timestamp4WithoutTimeZoneType() throws Exception {
   protected void timestamp3WithoutTimeZoneType() throws Exception {
     final Schema schema = connection.getTableSchema(null, null, "adbc_alltypes");
     assertThat(schema.findField("timestamp_without_time_zone_p3_t").getType())
-        .isEqualTo(new ArrowType.Timestamp(TimeUnit.MICROSECOND, null));
+        .isEqualTo(new ArrowType.Timestamp(TimeUnit.MILLISECOND, null));
```

Comment:
I'm not sure how this change is related to the CheckerFramework.

Reply:
I noticed there were two different type mapping functions and unified on the more comprehensive one.

Reply:
Since I didn't want to null-annotate two copies of basically the same function.